src/ui/server/controllers/insurance/business/turnover/index.test.ts

```diff
@@ -137,6 +142,19 @@ describe('controllers/insurance/business/turnover', () => {
         const expected = `${INSURANCE_ROOT}/${mockApplication.referenceNumber}${BROKER_ROOT}`;
         expect(res.redirect).toHaveBeenCalledWith(expected);
       });
+
+      it('should call mapAndSave.companyDetails once with natureOfBusiness and application', async () => {
```

Comment:
typo(s) here seems it should be `mapAndSave.turnover`and without `natureOfBusiness`?